ccpp/data/GFS_typedefs.meta

```diff
@@ -6940,16 +6940,16 @@
   kind = kind_phys
 [dkt]
   standard_name = atmosphere_heat_diffusivity
-  long_name = diffusivity for heat
+  long_name = atmospheric heat diffusivity
   units = m2 s-1
-  dimensions = (horizontal_loop_extent,vertical_dimension_minus_one)
+  dimensions = (horizontal_loop_extent,vertical_dimension)
   type = real
   kind = kind_phys
 [dku]
   standard_name = atmosphere_momentum_diffusivity
-  long_name = diffusivity for momentum
+  long_name = atmospheric momentum diffusivity
```

Comment:
@climbfuji Would you please also change the descriptions of dkt and dku with the same long name in ccpp/driver/GFS_diagnostics.F90?

Reply:
Good point, done.